src/content/experimental/ui-shell/usage.md

```diff
@@ -114,7 +163,11 @@ A Category must contain at least 3 L3 Links.
 
 -->
 
-## Panel Styles
+## Panels 
```

Comment:
Is this section really needed? Can we nest this info into the other area where we talk about header panels etc? Or more subtle cross link to it in places where we mention flexible, fixed or floating panels?